### DISCUSSION TITLE
Request for integration with Oracle networks.

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#13] [Integration]
As requested by the msgport and dApps consumers in Darwinia2, there are strong demands for integration with Oracles networks, for them to support our Darwinia2 chain, connect our chain to their platform and their oracle nodes can publish/supply data and the contracts in Darwinia2 can consume.

For reference:

https://ethereum.org/en/developers/docs/oracles/

Ran and Aki may already be following on this, anyone who would like to collaborate on this or sell your oracle services, please leave a comment on this thread.

**Comment by 1022ranji** [comment]
I'm on it.
<img width="822" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/21a58860-4161-4423-ad27-6939a2a8ebf9">

**Reply by SasoLithops** [reply]
Hi team!

I can only comment on Band Protocol. Astar and Moonbeam use it. Please check from them what their expirience is. 
I am sure it is much cheaper then Chainlink😉
But some of the people from Cosmos Eco I speak with don't give it much value🤷
I can ask around there more if Band is among chosen ones...

Best of luck choosing.

**Reply by 1022ranji** [reply]
Hi Saso! I tried to contact the Band team via dev@bandprotocol.com, but haven't received their reply for now. And I try to contact them via Discord, and now also waiting for their reply. I do want to know their fees and corresponding services 😄😄

Apart from Band, I have also had initial contact with several Oracle projects. So far, I've received replies from PYTH, UMA, Tellor, and DIA. However, there is still some way to go before final integration. 🏄

Actually, it's not limited to just one support. It would be great to have multiple Oracle projects supporting Darwinia Chain. 💪

**Reply by SasoLithops** [reply]
Hi Ran Ji😀

I will try on their telegram for them to reach to you. Although this slow response from them is no good at all. I will also ask around Cosmos why they not so valued.

Very good decision to have more🧠 💯

**Reply by SasoLithops** [reply]
Ran Ji I got this from them on TG. Posted in our ambassador TG also😊


You can reach out to @Shine#7393 and  @bun_band#0566  on the discord server

**Reply by 1022ranji** [reply]
Thank you Saso! have reached out to them on Discord!

**Reply by 1022ranji** [reply]
They gave us a form, we need to fill out the form first so Band can better understand what Darwinia needs. 😊😊
